src/AWS/CloudWatchLogs/CloudwatchLogs.purs

```diff
@@ -0,0 +1,67 @@
+module AWS.CloudwatchLogs where
+
+import Prelude
+import AWS.Core (AccessKeyId(..), Region(..), SecretAccessKey(..), SessionToken(..))
+import Control.Promise (Promise)
+import Control.Promise as Promise
+import Data.Bifunctor (bimap)
+import Data.Either (Either)
+import Data.Foldable (fold)
+import Data.Function.Uncurried (Fn1, runFn1)
+import Data.List.Types (NonEmptyList)
+import Data.Maybe (Maybe)
+import Data.Newtype (un)
+import Data.Nullable (Nullable, toNullable)
+import Effect (Effect)
+import Effect.Aff (Aff)
+import Effect.Class (liftEffect)
+import Foreign (ForeignError)
+import Simple.JSON (readJSON)
+
+foreign import data CloudwatchLogs :: Type
+
+type InternalMakeClientParams
+  = { region :: String
+    , secretAccessKey :: String
+    , accessKeyId :: String
+    , sessionToken :: Nullable String
+    }
+
+foreign import makeClientImpl :: Fn1 InternalMakeClientParams (Effect CloudwatchLogs)
+
+makeClient :: Region -> AccessKeyId -> SecretAccessKey -> Maybe SessionToken -> Effect CloudwatchLogs
+makeClient r a s t =
+  makeClientImpl
+    { region: (un Region r)
+    , secretAccessKey: (un SecretAccessKey s)
+    , accessKeyId: (un AccessKeyId a)
+    , sessionToken: toNullable $ map (un SessionToken) t
+    }
+
+type InternalGetDescribeLogGroupsOutput
+  = { logGroups ::
+        Array
+          { arn :: String
+          , creationTime :: Number
+          , kmsKeyId :: String
```

Comment:
Do we need kmsKeyId and metricFilterCount? If not can we just skip it there?

Reply:
To get a complete API I would prefer to keep it especially because this lib is open source.

Reply:
Ok fair point. Did we follow the same principle for the other API calls?